Doc/library/weakref.rst

```diff
@@ -492,7 +492,7 @@ is still alive.  For instance
     >>> obj = Object()
     >>> weakref.finalize(obj, print, "obj dead or exiting")  #doctest:+ELLIPSIS
     <finalize object at ...; for 'Object' at ...>
-    >>> exit()                                               #doctest:+SKIP
+    >>> del obj
     obj dead or exiting
```

Comment:
Ah, this change fix doctest, but this example doesn't match to the document...
I want to ignore this document block in doctest.